### DISCUSSION TITLE
Zipato keyboard changes state on its own.

### DISCUSSION THREAD

**Discussion by OUARZA** [zwave-js/zwave-js-ui#3193] [Q&A] [answered]
Hello,
Many of us have encountered a problem with the Zipato keyboard since using Zwave-JS.
I have the impression that when the equipment wakes up (at regular intervals), the last command played (passing a badge for example), is replayed even though no one has passed the badge.
Can you analyze the situation?
V: 8.19.0
THANKS

**Comment by robertsLando** [comment]
Are you using docker as deploy method? Could you try using `master` tag instead of `latest` ?

**Comment by OUARZA** [comment]
Good morning,
I use Jeedom. https://doc.jeedom.com/fr_FR/plugins/automation%20protocol/zwavejs/
I upgraded to ZwaveJS UI Version 8.22.0 and still have the same problem.
I have to delete the wakeup to prevent the last action from being replayed when I wake up.
Matthew

**Comment by robertsLando** [comment]
@AlCalzone Are you aware of issues like this on driver? @OUARZA could you also try with latest 8.22.3?

**Comment by OUARZA** [comment]
@robertsLando 
No, Jeedom has not yet integrated it into its plugin in beta version
https://doc.jeedom.com/en_US/plugins/automation%20protocol/zwavejs/beta/changelog

Here is a thread where several people have the problem.
https://community.jeedom.com/t/clavier-zipato-benext-bug-depuis-la-derniere-maj/106207/38

**Comment by AlCalzone** [comment]
@OUARZA can you report this as an issue in https://github.com/zwave-js/node-zwave-js, including driver logs that include a wakeup?
I may have an idea what's going on, but I need to confirm.

**Reply by OUARZA** [reply, marked as answer]
https://github.com/zwave-js/node-zwave-js/issues/6143
